sphinx/source/members/common_member_operations.rst

```diff
@@ -107,7 +107,7 @@ Once the proposal to recover the network has passed under the rules of the :term
             Node 3-->>Users: Response
         end
 
-Once the recovery of the private ledger on all the nodes that have joined the new network is complete, the ledger is fully recovered and users are able to continue issuing business transactions.
+Once the recovery of the private ledger is complete on all the nodes that have joined the new network, the ledger is fully recovered and users are able to continue issuing business transactions.
```

Comment:
This is inaccurate, the recovery of the private ledger only needs to have globally committed/completed on a consensus quorum of nodes (f+1 for raft and 2f+1 for pbft).

Reply:
I'll update it